functions-and-operators/string-functions.md

```diff
@@ -416,10 +416,42 @@ Return a string such that for every bit set in the value bits, you get an on str
 
 Return the index (position)of the first argument in the subsequent arguments.
 
+```sql
+SELECT FIELD('needle', 'A', 'needle', 'in', 'a', 'haystack');
+```
+
+```
++-------------------------------------------------------+
+| FIELD('needle', 'A', 'needle', 'in', 'a', 'haystack') |
++-------------------------------------------------------+
+|                                                     2 |
++-------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+Here the first argument of `FIELD()` is `needle` and this matches the second one of the following arguments, so the function returns 2.
```

Comment:
````suggestion
Return the index (position) of the first argument in the subsequent arguments.

In the following example, the first argument of `FIELD()` is `needle`, and it matches the second argument in the following list, so the function returns `2`.

```sql
SELECT FIELD('needle', 'A', 'needle', 'in', 'a', 'haystack');
+-------------------------------------------------------+
| FIELD('needle', 'A', 'needle', 'in', 'a', 'haystack') |
+-------------------------------------------------------+
|                                                     2 |
+-------------------------------------------------------+
1 row in set (0.00 sec)
```
````